CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 
 - Remove unused errors from validation package.
+- Do not set `config-controller.giantswarm.io/version` label to "0.0.0" on created App CRs.
```

Comment:
I think we should also remove setting pause annotation feature since it's set by the webhook.